### DISCUSSION TITLE
Gallery creating extra image at the start

### DISCUSSION THREAD

**Discussion by jordonsoh** [stashapp/stash#4892] [Q&A]
Whenever I scan galleries in folders, Stash creates a new image at the start of the gallery that I have to delete 1 by 1.

Is there any workaround this?

**Comment by WithoutPants** [comment]
I'm not sure what workaround you're after, and there's not a lot of information here. Are you needing a quicker way to delete the image or are you wanting to prevent the image from being created in the first place?

**Reply by jordonsoh** [reply]
Either solution would work for me. However, if you are able to provide me with both solutions that would be great. I would prefer preventing the image from being created, so I wouldn't have to deal with this issue for future files. But if there is a way to quickly delete all of the first image (generated image?) in every gallery, that would be helpful incase.
